qemu/deps/tdx/pccsadmin_test.sh

```
@@ -0,0 +1,138 @@
+if [ ! -f platform_collaterals.json ]; then
+    echo "Pccsadmin fetch failed, platform_collaterals.json was not generated."
+    exit 1
+fi
+
+# pccsadmin fetch
+expect -c "
+set timeout 240
+spawn pccsadmin fetch
+expect {
+    -re \"ApiKey.*:\" {
+        send \"$env(PCCS_PRIMARY_API_KEY)\r\"
```

Shellcheck (0.11.0)

[warning] env is referenced but not assigned (for output from commands, use "$(env ...)" ).

(SC2154)